galata/test/jupyterlab/menus.test.ts

```diff
@@ -176,3 +176,180 @@ function reduceItem(
   }
   return null;
 }
+
+test.describe('Top menu keyboard navigation', () => {
+  test.use({ autoGoto: false });
+  test('navigate to open file with keyboard', async ({ page }) => {
+    await page.goto();
+    for (let i = 0; i < 3; i++) {
+      await page.keyboard.press('Shift+Tab');
+    }
+    await page.keyboard.press('Enter');
+
+    expect(await page.menu.isOpen('File')).toBeTruthy();
```

Comment:
This test feels a bit brittle to me. We can make the test less brittle by not counting the number of tab presses needed, instead pressing tab continually until we land on the "File" menu bar item. Note that I did not test the following code suggestion:

```suggestion
    const fileMenu = page.getByRole('menuitem', { name: 'File' });
    while (true) {
      await page.keyboard.press('Shift+Tab');
      if (fileMenu.evaluate(el => el === document.activeElement)) {
        break;
      }
    }
    await page.keyboard.press('Enter');

    expect(await page.menu.isOpen('File')).toBeTruthy();
```